tests/fixtures/js-stubs/metricsMeta.js

```diff
@@ -0,0 +1,71 @@
+export function MetricsMeta(additionalMeta = []) {
+  return [
+    {
+      name: 'sentry.sessions.session',
```

Comment:
we don't usually do it, but I was wondering if we could use the created ENUMS here

Reply:
nope..its `.js ` never mind